clang/include/clang/Basic/Attr.td

```diff
@@ -1217,6 +1217,14 @@ def CoroDisableLifetimeBound : InheritableAttr {
   let SimpleHandler = 1;
 }
 
+def CoroAwaitElidable : InheritableAttr {
```

Comment:
does this also warrant an entry in `clang/docs/ReleaseNotes.rst`?

Reply:
I am adding this 
```
- Introduced a new attribute ``[[clang::coro_await_elidable]]`` on coroutine return types
  to express elideability at call sites where the coroutine is co_awaited as a prvalue.
```

wdyt? It's rather succinct but I prefer to leave more details in `AttrDocs.td`.